instrumentation/opentelemetry_grpcbox/src/opentelemetry_grpcbox.app.src

```diff
@@ -0,0 +1,14 @@
+{application, opentelemetry_grpcbox,
+ [{description, "grpcbox interceptor for OpenTelemetry instrumentation"},
+  {vsn, "0.1.0"},
+  {registered, []},
+  {applications,
+   [kernel,
+    stdlib
+   ]},
+  {env,[]},
+  {modules, []},
+
+  {licenses, ["Apache 2.0"]},
```

Comment:
```suggestion
  {licenses, ["Apache-2.0"]},
```
I'm not 100% sure but I think there's a demand to be using no whitespace in hex now.

Reply:
Yup, thanks, I copied this app from an older app -- rebar3 templates have been updated -- which is why numerous things in the .app are wrong.